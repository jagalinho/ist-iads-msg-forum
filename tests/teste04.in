A 123 Mensagem de teste
A 123 Outra mensagem de teste
A 567 Ainda outra mensagem de teste
A 890 A roda do autocarro roda roda
C roda
C carro
C auto
A 890 A roda do autocarro roda roda roda
A 890 A roda do autocarro roda
A 123 A roda do autocarro roda
C roda
C carro
C auto
X