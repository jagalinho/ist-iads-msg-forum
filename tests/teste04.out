*WORD roda:3
*WORD carro:0
*WORD auto:0
*WORD roda:11
*WORD carro:0
*WORD auto:0
7
